botorch/models/approximate_gp.py

```diff
@@ -67,6 +67,14 @@
 
 
 class ApproximateGPyTorchModel(GPyTorchModel):
+    """
```

Comment:
```suggestion
    r"""
```

Reply:
We don't need to manually check this now that we have D301 enabled, which will error if there's a backslash in a non-raw docstring